app/services/chart_builder.rb

```diff
@@ -1,5 +1,38 @@
 class ChartBuilder
-  def initialize(benchmark_runs)
+  
+  # @data[:columns] is JSON that looks like [{ name: "benchmark1", data: [1.1, 1.2] }]
+  # @data[:categories] is a an array of version hashes: 
+  # [
+  #   {
+  #     version: "0",
+  #     environment: "ruby 2.2.0dev"
+  #   },
+  #   {
+  #     version: "0",
+  #     environment: "ruby 2.2.0dev"
+  #   }
+  # ]
+  attr_accessor :data
```

Comment:
If you know that the only two keys will be `columns` and `categories`, then why not 

```
attr_accessor :columns, :categories
```

You'd usually like to avoid variable and method names like `data`. Everything in the computer world is data and such names are not telling you what do they refer to.